api/schemas/catalog.json

```diff
@@ -94,6 +94,30 @@
             "items": {
                 "type": "string"
             }
+        },
+        "metadata": {
+            "description": "A mapping of non-recursive File resources to their required metadata",
+            "type": "object",
+            "patternProperties": {
+                "^.+$": {
```

Comment:
I couldn't get `"$ref"` to work, but I really want this to refer to file_metadata.json.